draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -2123,42 +2124,82 @@ Sun Nov  6 08:49:37 1994         ; ANSI C's asctime() format
 </t>
 </section>
 
-<section title="Message Payload" anchor="payload">
+<section title="Payload" anchor="payload">
 <iref item="payload"/>
 <t>
-   Some HTTP messages transfer a complete or partial representation as the
-   message "<x:dfn>payload</x:dfn>".  In some cases, a payload might contain
-   only the associated representation's header fields (e.g., responses to
-   HEAD) or only some part(s) of the representation data
-   (e.g., the <x:ref>206 (Partial Content)</x:ref> status code).
+   HTTP messages often transfer a complete or partial representation as the
+   message "<x:dfn>payload</x:dfn>".  The payload data is transferred as a
+   stream of octets, after the header section, and delineated by message
+   framing.
```

Comment:
```suggestion
   HTTP messages often transfer a complete or partial representation as the
   message "<x:dfn>payload</x:dfn>", consisting of a header section, <x:dfn>payload data</x:dfn>,
   and one or more trailer sections.  The payload data is transferred as a
   stream of octets, after the header section, and delineated by message
   framing.
```

Reply:
I don't understand this suggestion because it says that the trailers are part of the payload, but they are not (they are just part of the transfer encoding in HTTP/1.1).

I think what you are looking for is a better definition of payload, right?

Reply:
I rewrote the definition.